python/pyspark/sql/context.py

```diff
@@ -253,6 +254,8 @@ def createDataFrame(self, data, schema=None, samplingRatio=None):
            If it's not a :class:`pyspark.sql.types.StructType`, it will be wrapped into a
            :class:`pyspark.sql.types.StructType` and each record will also be wrapped into a tuple.
 
+           Added verifySchema.
```

Comment:
+1. I wasn't aware of this, but it looks like it's possible to have multiple `versionchanged` directives in the same docstring.